runhouse/resources/hardware/cluster.py

```diff
@@ -2271,8 +2271,10 @@ def create_process(
         env_vars: Optional[Dict] = None,
         compute: Optional[Dict] = None,
         conda_env_name: Optional[str] = None,
+        runtime_env: Optional[Dict] = None,
```

Comment:
I don't think we want to pass runtime_env as an arg, I think I can just fix the create replica code